apis/swagger.yml

```diff
@@ -895,18 +895,10 @@ paths:
       operationId: "ContainerCheckpointCreate"
       parameters:
         - $ref: "#/parameters/id"
-        - name: "checkpointdir"
-          in: "query"
-          description: "checkpoint directory"
-          type: "string"
-        - name: "checkpointid"
-          in: "query"
-          description: "checkpoint id"
-          type: "string"
-        - name: "exit"
-          in: "query"
-          description: "exit after checkpoint"
-          type: "bool"
+        - name: "body"
+          in: "body"
+          schema:
+            $ref: "#/definitions/CheckpointCreateOptions"
```

Comment:
Could you do this kind of change in another pull request? Since then swagger.yml refactoring has nothing to do with the swagger upgrade. WDYT? @zhuangqh

Reply:
> Could you do this kind of change in another pull request? Since then swagger.yml refactoring has nothing to do with the swagger upgrade. WDYT? @zhuangqh

Actually, this part of code is wrong, which failed to pass the stricter validation. Futhermore, it makes this PR fail to pass the CI. So this part of change should be included in this PR although it looks confused...

Reply:
Roger that.